src/Symfony/Component/Asset/VersionStrategy/JsonManifestVersionStrategy.php

```diff
@@ -39,7 +39,7 @@ public function __construct(string $manifestPath, HttpClientInterface $httpClien
         $this->manifestPath = $manifestPath;
         $this->httpClient = $httpClient;
 
-        if (null === $this->httpClient && 0 === strpos(parse_url($this->manifestPath, \PHP_URL_SCHEME), 'http')) {
+        if (null === $this->httpClient && ($scheme = parse_url($this->manifestPath, \PHP_URL_SCHEME)) && 0 === strpos($scheme, 'http')) {
```

Comment:
What about `0 === strpos(parse_url($this->manifestPath, \PHP_URL_SCHEME) ?? '', 'http')`?

Reply:
That would work as well. In earlier reviews for similar changes, @nicolas-grekas wanted me to avoid passing empty strings to string functions.

Reply:
I'm fine either ways here :)

Reply:
Let's leave the line as-is then.